README.md

```diff
@@ -111,123 +72,83 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Comming Soon 🎈]()
```

Comment:
- [ ] kindly add a live link to this project

Reply:
Thanks for the feed back